src/components/results/securityanalysis/security-analysis-result-tab.tsx

```diff
@@ -265,6 +266,11 @@ export const SecurityAnalysisResultTab: FunctionComponent<
                     <SecurityAnalysisResultN
                         result={result}
                         isLoadingResult={isLoadingResult}
+                        onSortChanged={onSortChanged}
+                        sortConfig={sortConfig}
+                        filterSelector={filterSelector}
+                        updateFilter={updateFilter}
+                        filterEnums={filterEnums as FilterEnums}
```

Comment:
I think it would be way better to type "useFetchFiltersEnums" return value instead of casting it here
```
const [result, setResult] = useState<FilterEnums>({
      computationsStatus: null,
      limitTypes: null,
      branchSides: null,
  });
```

Reply:
Done